pennylane/measurements/mid_measure.py

```diff
@@ -225,6 +225,7 @@ def __init__(
         postselect: Optional[int] = None,
         id: Optional[str] = None,
     ):
+        self.batch_size = None
```

Comment:
I imagine this is strictly here because the device at some point checks the batch size of everything in the tape operations?

Reply:
Yes.